frontend/cypress/support/commands.js

```diff
@@ -11,12 +11,12 @@
 //
 // -- This is a parent command --
 Cypress.Commands.add('login', () => {
-  const login = 'marikoa';
-  const password = 'MtXzwmLYTDq5Gucr';
+  const login = 'marikotest';
+  const password = 'aquarium';
 
   cy.request({
     method: 'POST',
-    url: `http://localhost:3001/api/v3/user/sign_in?login=${login}&password=${password}`,
+    url: `http://localhost:3001/api/v3/token/create?login=${login}&password=${password}`,
```

Comment:
Shouldn't this use the `baseURL`?

Reply:
Yes, we can definitely do that!

Reply:
Turns out I'll need to do a bit more work to parameterize the front and back end separately.